src/Mod/PartDesign/App/FeatureLoft.cpp

```diff
@@ -110,13 +110,24 @@ App::DocumentObjectExecReturn *Loft::execute(void)
         for(TopoDS_Wire& wire : wires)
             wiresections.emplace_back(1, wire);
 
-        for(App::DocumentObject* obj : multisections) {
+        for(auto obj : multisections) {
             if(!obj->isDerivedFrom(Part::Feature::getClassTypeId()))
                 return  new App::DocumentObjectExecReturn("Loft: All sections need to be part features");
 
+            TopoDS_Shape shape;

```

Comment:
Please add comments why you do something. Now it is clear, but what about in 2 years when another person looks at it?